### PR TITLE
Animated cards

### PR DESCRIPTION
**Pull request by PragyaTripathi990**
Thank you for making a pull request to ZTM's Animation Nation project. We will review your pull request as soon as we can, but keep in mind we can get very busy during Hacktoberfest.

While you're waiting, please look over this checklist. Feel free to write a description of what you did as well!

## What I did

I have have created cards that shows animation on hover. I hope you would merge my PR.

## Pull request checklist

- [x] This pull request uses **HTML and CSS** only and provides a `meta.json` file.
- [x] All files in this pull request are in a directory named: `<github_username>-<art_name>` without the <>.
- [x] Your pull request contains `index.html` and `styles.css`.
	- [x] Your html file is named exactly: `index.html`
	- [x] Your css file is named exactly: `styles.css`
- [x] Your pull request your `meta.json` with:
	- [x] `githubHandle`: being your unique github user name,
	- [x] `artName` : name of your animation
- [x] Your submission includes **at least one animation**.
- [x] Your pull request **does not modify any other files** in the repository.

#### Not normally needed
- [ ] Check this if you would like to discuss an improvement besides adding animations

**Note: if your pull request makes an improvement besides adding a CSS animation you will need to discuss this with the maintainers. You can help kickstart this process by explaining what your change does and why you are making it in the "What I did" section**

## What to expect

- If your pull request meets the checklist requirements it will be approved and merged by a maintainer
- If your pull request needs some changes. A maintainer will leave a comment requesting changes
- If your pull request significantly violates the criteria in the checklist it will be closed with feedback.

### CODE REVIEW

File: Art/PragyaTripathi990-cardsAnimation/styles.css
Comment (LaurelineP) [inline Art/PragyaTripathi990-cardsAnimation/styles.css:1]: ## CSS comments 
Please removed the commented CSS rules

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Aloha and thanks for your contribution 🙌
Before we could merge your PR please address the given feedbacks :
- Please remove the commented css rules
- your contribution is missing a `meta.json` file ( see the README.md file )  :)

File: Art/PragyaTripathi990-cardsAnimation/styles.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    background-color: rgb(249, 216, 216);
+    border-radius: 12px;
+    padding: 30px;
+    padding-top: 70px;
+    display: flex;
+     align-items: center; 
+    /* justify-content: center; */
+     /* flex-direction: column; 
+     color: gray
+}  */
```
Comment (LaurelineP) [inline Art/PragyaTripathi990-cardsAnimation/styles.css:59]: - [ ] There are still commented blocks you could remove here

File: Art/PragyaTripathi990-cardsAnimation/styles.css [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+ button:hover{
+  
+ box-shadow: 0 6px 12px rgba(0, 0, 0, 0.6); 
+ transform: scale(1.2);
+} 
+.box:hover{
+    color:white;
+    /* background: linear-gradient(135deg, #8A2BE2, #A0522D) */
+    background: linear-gradient(135deg, #9B59B6, #4A148C);
+    /* background: linear-gradient(135deg, #8A2BE2, #DA70D6); */
```
Comment (LaurelineP) [inline Art/PragyaTripathi990-cardsAnimation/styles.css:95]: - [ ] There are still commented lines you could remove here :)

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Also, you need to add a `meta.json`  you could read about here 
[README.md]([Code](https://github.com/zero-to-mastery/Animation-Nation))
as you can see in you PR template 
- [ ] Missing `meta.json` : please add your `meta.json` file in the folder :)

**Comment by PragyaTripathi990** [comment]
@LaurelineP Hey, I am done with the requested changes! Can you please review it.

**Pull request merged**